package-lock.json

```diff
@@ -1,6 +1,2322 @@
 {
```

Comment:
This entire file should be deleted along with the `package.json` in the root directory (`package-lock.json` and `package.json` should only be in the `api/`, `frontend/`, and `usb/` directories